internal/bminventory/inventory.go

```diff
@@ -4275,6 +4301,10 @@ func (b *bareMetalInventory) updateInfraEnvData(ctx context.Context, infraEnv *c
 		}
 		if staticNetworkConfig != infraEnv.StaticNetworkConfig {
 			updates["static_network_config"] = staticNetworkConfig
+
+			if err = b.setStaticNetworkUsage(db, infraEnv.ClusterID, staticNetworkConfig); err != nil {
+				log.WithError(err).Warnf("failed to set static network usage for cluster %s", infraEnv.ClusterID)
```

Comment:
Not that I want us to do something different than for the other feature-usage handling, but it's a bit weird to just log the errors instead of returning them, no?

Reply:
the feature usage is internal and only for statistics so I don't think we should throw exception and not allow the user to create or update cluster in this case

Reply:
I'm just a bit worried about the debugability impacts of our service
Either it's a regular user or a QE tester, you rather have your problems visible than invisible

I don't want to change it the scope of this PR, but really it's mostly downsides ignoring errors in web services